### PR TITLE
OOPDEF-61: Added "Review" page.

### PR DESCRIPTION
**Pull request by harrymaynard-maximus**
- [x] After these changes, the app was run and still works as expected
- [x] Tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### CODE REVIEW

File: oop/src/helpers/date.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { format } from 'date-fns';
+
+export const formatDate = (date) => {
+  if (!date) {
+    return null;
+  }
+  return format(date, 'MMMM d, y')
+};
```
Comment (cyrilbravo) [inline oop/src/helpers/date.js:8]: Is it possible to put this on DateInput.vue since it's also importing from 'date-fns' or should it be a separate file?

**Review by cyrilbravo — COMMENTED** [review]
Aside from the comment below, Review Page and adding edit icon looks good. Nice job Harry!

File: oop/src/helpers/date.js (hunk 1 above)
Comment (harrymaynard-maximus) [reply oop/src/helpers/date.js:8]: This is a helper method, so I think it's best to keep it separate from the `DateInput` component.

**Review by cyrilbravo — APPROVED** [review]
Talked to Harry about the comment.

**Pull request merged**